Analytics.Xamarin.Pcl/Client.cs

```diff
@@ -155,9 +117,9 @@ public virtual void Initialize(string writeKey, Config config)
 		/// Pass in values in key-value format. String key, then its value
 		/// { String, Integer, Boolean, Double, or Date are acceptable types for a value. } </param>
 		///
-		public void Identify(string userId, IDictionary<string, object> traits)
+		public async Task Identify(string userId, IDictionary<string, object> traits)
 		{
-			Identify(userId, traits, null);
+			await Identify(userId, traits, null);
```

Comment:
On these ones where it is a wrapper method to call an internal or more specific overload of the method, you can just return from them instead of awaiting them.

e.g. `return Identify (userId, traits, null)`.  What this does since the internal one also returns a Task object is you just grab that object and return it yourself, which is then awaited by the caller.  This reduces overhead from multiple synchronization contexts when the same action would still occur.  This would also allow you to drop the `async` keyword from the wrapper method.  Though since it returns a `Task` object should still be named with `Async` at the end like mentioned in the comment above.